sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/text/TextFileFormat.scala

```diff
@@ -40,6 +40,8 @@ class TextFileFormat extends TextBasedFileFormat with DataSourceRegister {
 
   override def shortName(): String = "text"
 
+  override def toString: String = shortName.toUpperCase
```

Comment:
As you might already know, I see [CSVFileFormat.scala#L43](https://github.com/apache/spark/blob/cb1b9d34f37a5574de43f61e7036c4b8b81defbf/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVFileFormat.scala#L43) and [JsonFileFormat](https://github.com/apache/spark/blob/abff92bfdc7d4c9d2308794f0350561fe0ceb4dd/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/json/JsonFileFormat.scala#L144) use string value `"CSV"` and `"JSON"`  rather then using `shortName.toUpperCase` for this case. It might be great if they are matched up.